Makefile

```diff
@@ -329,14 +329,14 @@ pre-commit:
 	pre-commit run --files ./src/assisted_test_infra/test_infra/* ./src/tests/*
 
 _reformat:
-	black $(LINT_CODE_STYLING_DIRS) --line-length=120
-	isort $(LINT_CODE_STYLING_DIRS) --profile=black --line-length=120
+	black . --line-length=120
```

Comment:
should the lint include these new targets and fail if changes are detected?

Reply:
That's a good question
Checking

Reply:
OK, so it seems like files in ``scripts/`` dir are not being checked by ``flake8``
So doing ``make reformat`` will fix all lint errors (which is a required behavior), but pep8/flake8/black are not enforced for ``scripts/`` dir (which is a desired behavior)
I'll unhold this PR and check how to include this dir as well (maybe ``flake8`` only considers packages defined in ``pyproject.toml``? I still unsure about the behavior here)